### PR TITLE
ddl: Support the operation of dropping multi-indexes in one statement

### PR DESCRIPTION
**Pull request by ou-bing**
What problem does this PR solve?
#14765 

What is changed and how it works?
dropping multi-indexes in one statement:

```
ALTER TABLE tbl_name
    [alter_specification [, alter_specification] ...]
```

```
alter_specification:
| DROP [INDEX] index_name
Check List
```

### CODE REVIEW

**Comment by bb7133** [comment]
Hi @ou-bing , is this PR still WIP?

File: ddl/ddl_api.go [hunk 1]
```diff
@@ -4778,6 +4781,93 @@ func (d *ddl) DropIndex(ctx sessionctx.Context, ti ast.Ident, indexName model.CI
 	return errors.Trace(err)
 }
 
+func (d *ddl) DropIndexes(ctx sessionctx.Context, ti ast.Ident, specs []*ast.AlterTableSpec) error {
```
Comment (bb7133) [inline ddl/ddl_api.go:4784]: This function is quite similar to `DropIndex`, can we combine them together and use `DropIndexes` for dropping single index as well?

**Comment by ou-bing** [comment]
> Hi @ou-bing , is this PR still WIP?

yes.this function has not been completed yet.😂

File: ddl/ddl_api.go (hunk 1 above)
Comment (ou-bing) [reply ddl/ddl_api.go:4841]: 👌

**Comment by bb7133** [comment]
> > Hi @ou-bing , is this PR still WIP?
> 
> yes.this function has not been completed yet.

I see, good luck! BTW I've added a 'Release Note' part to your PR description to avoid 'no release note complaint' of the bot.

**Comment by ou-bing** [comment]
> > > Hi @ou-bing , is this PR still WIP?
> > 
> > 
> > yes.this function has not been completed yet.
> 
> I see, good luck! BTW I've added a 'Release Note' part to your PR description to avoid 'no release note complaint' of the bot.

thank you，i will add it in next "draft pr".

**Pull request closed without merging**